app/main/posts/savedsearches/mode-context.html

```diff
@@ -110,7 +110,7 @@ <h1 class="mode-context-title">
         </div>
         <!--// END IF //-->
         <div class="form-field">
-            <a ui-sref="list({view: collectionView})" type="button" class="button-link">
+            <a ui-sref="list.map" ui-sref-opts="{reload: true}" type="button" class="button-link">
```

Comment:
why do we have to specify reload here? (and in the other places we're using it?)

Reply:
Because you want to force a reload of the data, else you don't get the updated lists